### ISSUE TITLE
[A11y] VoiceOver doesn't annouce instructions for a selected annotation

### ISSUE THREAD

**Issue by Dananji** [samvera-labs/ramp#815]
### Description

When using keyboard navigation along with VoiceOver in the transcript component, it announces the cue content and instructions to activate a selected cue. The annotations component follows a similar pattern and is expected yield similar behavior, but that seems to be not the case.

Comment highlighting this issue: https://github.com/samvera-labs/ramp/pull/805#issuecomment-2867578837 by Emily;

> When I tab into the Annotation list from the menu and use the up/down arrow keys to select an annotation, voiceover reads it at least twice in a row. That's problematic.
    In the transcript tab, when I use the up/down arrow keys to select a cue, voiceover reads it once and then provides an instruction to use Ctrl + Option + Enter to select it. Doing that seeks to that cue in the player. It looks like the same keyboard combination works to seek to the beginning timepoint for an Annotation in the annotations tab, but voiceover doesn't announce the instructions.

### Done Looks Like
- [ ] VoiceOver reads out content and instructions relevant to a selected annotation row similar to a transcript cue
- [ ] Keyboard navigation works as expected for annotations component (https://github.com/samvera-labs/ramp/issues/781#issuecomment-2806990134)

**Comment by Dananji** [comment]
Changes made to fix this issue:
- use `role="button"` on the focus-able portion of the annotation row, which enables the instruction announcement for activation of the focused annotation by a screen reader

This is waiting on #805 to be reviewed and merged.

**Comment by Dananji** [comment]
This is an easy fix. I can make this change and publish a patch release of Ramp.
